common/src/main/java/com/wynntils/features/inventory/ItemFavoriteFeature.java

```diff
@@ -33,22 +35,49 @@ public class ItemFavoriteFeature extends Feature {
     @RegisterConfig
     public final HiddenConfig<Set<String>> favoriteItems = new HiddenConfig<>(new TreeSet<>());
 
+    @RegisterConfig
+    public final Config<Integer> lootChestCloseOverride = new Config<>(3);
+
+    private int lootChestCloseOverrideCounter = 0;
+
     @SubscribeEvent
     public void onChestCloseAttempt(ContainerCloseEvent.Pre e) {
         if (!Models.WorldState.onWorld()) return;
         if (!Models.Container.isLootOrRewardChest(McUtils.mc().screen)) return;
 
+        boolean containsFavorite = false;
         NonNullList<ItemStack> items = ContainerUtils.getItems(McUtils.mc().screen);
         for (int i = 0; i < 27; i++) {
             ItemStack itemStack = items.get(i);
 
             if (isFavorited(itemStack)) {
-                McUtils.sendMessageToClient(Component.translatable("feature.wynntils.itemFavorite.closingBlocked")
-                        .withStyle(ChatFormatting.RED));
-                e.setCanceled(true);
-                return;
+                containsFavorite = true;
+                break;
             }
         }
+
+        if (!containsFavorite) return;
+
+        if (lootChestCloseOverride.get() == 0) {
+            McUtils.sendMessageToClient(Component.translatable("feature.wynntils.itemFavorite.closingBlocked")
+                    .withStyle(ChatFormatting.RED));
+            e.setCanceled(true);
+            return;
+        }
+
+        if (lootChestCloseOverride.get() < lootChestCloseOverrideCounter) return;
+        lootChestCloseOverrideCounter++;
```

Comment:
These two lines seem to be in the wrong order..? Have you tried that this work?

Reply:
Yes, it does. "If the current override counter is higher than the set minimum config, do not try to block closing".

Reply:
Ah, right, the  comparison was between the config max to the left and the actual counter to the right.

For some reason I found that counter-intuitive. Do you mind fliipping the comparison to a > instead?

Reply:
Did, no problem.